src/test/datascience/notebook/remoteNotebookEditor.vscode.test.ts

```diff
@@ -161,44 +161,11 @@ suite('Remote Kernel Execution', function () {
                 )}`
         );
     });
-
-    test('Remote kernels are removed when switching to local @kernelPicker', async function () {
-        await controllerLoader.loaded;
```

Comment:
This is not possible with the new and stable kernel pickers, this applies to the old kernel picker